hoodie-client/src/test/avro/SpecificRecordExampleValue.avsc

```diff
@@ -0,0 +1,8 @@
+{"namespace": "com.uber.hoodie.example",
```

Comment:
please consolidate under `src/test/resources` with other such schema..

Reply:
I've replicate the pattern implemented in hoodie-common/src/main/avro but for the test schema

Reply:
I understand.. but as far as tests go, its all in `src/test/resources`.. So prefer to just keep it in once place..

Reply:
the avro maven plugin doesn't take the setting and search in [ERROR] Failed to execute goal org.apache.avro:avro-maven-plugin:1.7.6:schema (default-cli) on project hoodie: neither sourceDirectory: /home/xjodoin/git/hudi/src/main/avro or testSourceDirectory: /home/xjodoin/git/hudi/src/test/avro are directories

Reply:
okay.. :)